doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3159,7 +3160,7 @@ the conversion in hypotheses :n:`{+ @ident}`.
 
    + A constant can be marked to be unfolded only if applied to enough
      arguments. The number of arguments required can be specified using the
-     ``/`` symbol in the argument list of the :cmd:`Arguments` vernacular command.
+     ``/`` symbol in the argument list of the :cmd:`Arguments <Arguments (implicits)>` vernacular command.
```

Comment:
Didn't know that this was possible. This is cool!

Reply:
Not really.  The problem is that this snippet is in fact the only documentation for the `/` feature (I think), so the best might be to link to the Arguments section of the docs as a whole.

Reply:
I think we should open an issue about this (and fix later).